packages/oauth-app/src/OAuthStatus.tsx

```diff
@@ -69,7 +69,7 @@ export const OAuthStatus: React.FunctionComponent = () => {
         else setProfile(userProfile)
 
         await userService.fetchUser()
-        const username = await userService.getUsername()
+        const username = (await userService.getUsername()) ?? '<not logged-in>'
```

Comment:
we use user named `public` when not logged in
https://github.com/hawtio/hawtio-next/blob/28016194918cea17fd5da50ca4a3c9b493d92271/packages/hawtio/src/auth/globals.ts#L6

Reply:
the default for `moduleResolution` is quite complicated:

> Node10 if [module](https://www.typescriptlang.org/tsconfig/#module) is CommonJS; Node16 if [module](https://www.typescriptlang.org/tsconfig/#module) is Node16, Node18, or Node20; NodeNext if [module](https://www.typescriptlang.org/tsconfig/#module) is NodeNext; Bundler if [module](https://www.typescriptlang.org/tsconfig/#module) is Preserve; Classic otherwise.

about `target`:

> hawtio/hawtio standalone still uses es5

right! but I changed to `esnext` in hawtio/hawtio-next/app/tsconfig.json.

I see no reason to support "old browsers" - no one uses them and the code is really hacky with `es5` - most (if not all for what we use) of the babel tricks are no longer needed in modern browsers